daemons/fenced/pacemaker-fenced.c

```diff
@@ -1272,6 +1272,7 @@ st_peer_update_callback(enum crm_status_type type, crm_node_t * node, const void
 
         crm_debug("Broadcasting our uname because of node %u", node->id);
         send_cluster_message(NULL, crm_msg_stonith_ng, query, FALSE);
+        send_cluster_message(NULL, crm_msg_crmd, query, FALSE);
```

Comment:
This is not needed; it would only be sent to other nodes' fencers (i.e. the members of the fencer CPG group), not their controllers.